_dashboards/admin-ui-index/index-management.md

```diff
@@ -51,7 +51,13 @@ To attach policies to indexes, perform the following steps:
 
 The **Indices** section displays a list of indexes in your OpenSearch cluster. For each index included, you can see its heath status (`green`, `yellow`, or `red`), policy (if the index is managed by a policy), status, total size, size of primaries, total documents, deleted documents, primaries and replicas.
 
-#### Create index
+Below are the health statuses an index can be in:
+
+- Green: All primary and replica shards are assigned.
+- Yellow: At least one replica shard is not assigned.
+- Red: At least one primary shard is not assigned.
+
+### Create index
```

Comment:
I'm assuming these headings reflect the names of operations, but if not, they should be in the form "Creating an index" (https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md#topic-titles).